### PR TITLE
Updated the main ReadME file with a small description of the web app …

### PR DESCRIPTION
**Pull request by Shoetan**
Closes:https://github.com/ArslanYM/StarterHive/issues/39

Updated the main ReadME file with a small description of the web app and a Link that leads to the web app directory.

Before:
Screenshot from 2023-05-22 14-09-24

After:
Screenshot from 2023-05-22 14-09-50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,12 @@ You can also contribute to this project if you are new to open source:
  </details>
 
 
+ ### 🏊 Jump In:
+StarterHive is an open source project that helps global new developers in contributing to beginner friendly issues.Accelerating skills dvelopment,
```
Comment (ArslanYM) [inline README.md:68]: This line would definitely need some changes. You can use CHATGPT to rephrase the sentences in a better way.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ You can also contribute to this project if you are new to open source:
...
+paving the way for inclusive open-source contribtion worldwide.
```
Comment (ArslanYM) [inline README.md:69]: These changes dont imply what the project is.

**Review by ArslanYM — CHANGES_REQUESTED** [review]
Definitely needs some rephrasing.  And clear cut description of the project

**Comment by Shoetan** [comment]
@ArslanYM Thank you so much for your feedback and guidance. I sincerely appreciate it. Regarding the changes, you would need, I need some more clarification on what the project is all about. From what I can gather StarterHive helps by eliminating the boring task of manually searching through repositories and looking for issues to solve.

**Comment by ArslanYM** [comment]
@Shoetan yes thats right. Starter Hive helps you find issues in projects or codebases which are run my companies/organisations that hire people remotely.

**Comment by Shoetan** [comment]
@ArslanYM Thank you once again for your patience and guidance. I have just made the changes you requested to close this issue.

**Comment by ArslanYM** [comment]
This PR has conflicts with the main branch, maybe because you didnt pull the updated changes. Sorry to say but this PR is closeworthy. Thankyou for your contribution

**Pull request closed without merging**